spec/feature/time_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "capybara/rspec"
+require_relative '../../app.rb'
+require 'date'
+
+feature 'adding time of last update' do
+  scenario 'include the time count last updated' do
+    visit('/')
+    click_button "Reset"
+    time = Time.now.round
+    expect(page).to have_content(time)
```

Comment:
time-dependent tests are always a bit brittle -- they may fail for unrelated reasons, e.g. if for some reason the test takes a little longer to run each separate command. 

I suppose it is an unlikely scenario that a whole second would go by but it's not impossible if the CPU of the machine running the tests gets stalled by other running processes. You could make this a bit less likely by rounding the time to minutes.

Another (better?) option is to use mocking/stubbing to find a way to control what time will be returned when you run your test. We usually don't mock/stub anything in feature tests but for timing-related tests, it is sometimes necessary to break that rule.